<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addborowerclerk</class>
 <widget class="QMainWindow" name="addborowerclerk">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit3">
    <property name="geometry">
     <rect>
      <x>314</x>
      <y>210</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="l1">
    <property name="geometry">
     <rect>
      <x>344</x>
      <y>120</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Student</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit1">
    <property name="geometry">
     <rect>
      <x>314</x>
      <y>150</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="l3">
    <property name="geometry">
     <rect>
      <x>254</x>
      <y>180</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Roll-No</string>
    </property>
   </widget>
   <widget class="QLabel" name="l2">
    <property name="geometry">
     <rect>
      <x>254</x>
      <y>150</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b1">
    <property name="geometry">
     <rect>
      <x>334</x>
      <y>240</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
   <widget class="QLabel" name="l4">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>210</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit2">
    <property name="geometry">
     <rect>
      <x>314</x>
      <y>180</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>300</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Enter a different roll no !</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
